{
	"alias": "FEDORA-2018-c82fc3e109",
	"autokarma": true,
	"bugs": [
		{
			"bug_id": 1540946,
			"title": "SELinux is preventing mysqld from using the 'sys_nice' capabilities.",
			"bugzilla": {
				"bug_id": "1540946",
				"creation_ts": "2018-02-01 11:38:05 +0000",
				"short_desc": "SELinux is preventing mysqld from using the 'sys_nice' capabilities.",
				"delta_ts": "2018-11-10 03:18:14 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"priority": "unspecified",
				"bug_severity": "unspecified",
				"depends_on": [
					"1593639",
					"1629041"
				],
				"blocked": [
					{
						"bug_id": "1628814",
						"error": "NotPermitted"
					},
					{
						"bug_id": "1628815",
						"error": "NotPermitted"
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "11241154",
						"comment_count": "0",
						"who": {
							"text": "louzaoh",
							"name": "Héctor Louzao"
						},
						"bug_when": "2018-02-01 11:38:05 +0000",
						"thetext": "Description of problem:\nmysql-workbench-community-6.3.10-1.fc27.x86_64\nmysql-community-common-8.0.4-0.1.rc.fc27.x86_64\nmysql-community-libs-8.0.4-0.1.rc.fc27.x86_64\nmysql57-community-release-fc27-10.noarch\nmysql-community-client-8.0.4-0.1.rc.fc27.x86_64\nmysql-community-server-8.0.4-0.1.rc.fc27.x86_64\n\nselinux-policy-targeted-3.13.1-283.21.fc27.noarch\nselinux-policy-3.13.1-283.21.fc27.noarch\n\nsetroubleshoot-server-3.3.15-1.fc27.x86_64\nsetroubleshoot-3.3.15-1.fc27.x86_64\nsetroubleshoot-plugins-3.3.9-3.fc27.noarch\n\nupdate to mysql 8.0.4\nSELinux is preventing mysqld from using the 'sys_nice' capabilities.\n\n*****  Plugin catchall (100. confidence) suggests   **************************\n\nSi cree que mysqld debería tener la capacidad de sys_nice de forma predeterminada.\nThen debería reportar esto como un error.\nPuede generar un módulo de política local para permitir este acceso.\nDo\npermita el acceso temporalmente ejecutando:\n# ausearch -c 'mysqld' --raw | audit2allow -M mi-mysqld\n# semodule -X 300 -i mi-mysqld.pp\n\nAdditional Information:\nSource Context                system_u:system_r:mysqld_t:s0\nTarget Context                system_u:system_r:mysqld_t:s0\nTarget Objects                Unknown [ capability ]\nSource                        mysqld\nSource Path                   mysqld\nPort                          <Desconocido>\nHost                          (removed)\nSource RPM Packages           \nTarget RPM Packages           \nPolicy RPM                    selinux-policy-3.13.1-283.21.fc27.noarch\nSelinux Enabled               True\nPolicy Type                   targeted\nEnforcing Mode                Enforcing\nHost Name                     (removed)\nPlatform                      Linux (removed) 4.14.14-300.fc27.x86_64 #1 SMP Fri\n                              Jan 19 13:19:54 UTC 2018 x86_64 x86_64\nAlert Count                   1095\nFirst Seen                    2018-02-01 10:46:40 CET\nLast Seen                     2018-02-01 12:36:37 CET\nLocal ID                      15c9c979-5c2f-4650-a27a-91e54915a9f9\n\nRaw Audit Messages\ntype=AVC msg=audit(1517484997.191:5218): avc:  denied  { sys_nice } for  pid=27326 comm=\"mysqld\" capability=23  scontext=system_u:system_r:mysqld_t:s0 tcontext=system_u:system_r:mysqld_t:s0 tclass=capability permissive=0\n\n\nHash: mysqld,mysqld_t,mysqld_t,capability,sys_nice\n\nVersion-Release number of selected component:\nselinux-policy-3.13.1-283.21.fc27.noarch\n\nAdditional info:\ncomponent:      selinux-policy\nreporter:       libreport-2.9.3\nhashmarkername: setroubleshoot\nkernel:         4.14.14-300.fc27.x86_64\ntype:           libreport"
					},
					{
						"isprivate": "0",
						"commentid": "11241496",
						"comment_count": "1",
						"who": {
							"text": "lvrabec",
							"name": "Lukas Vrabec"
						},
						"bug_when": "2018-02-01 13:17:06 +0000",
						"thetext": "Hi Mysql folks, \n\nDoes mysql using sys_nice capability? \n\n       CAP_SYS_NICE\n              * Raise process nice value (nice(2), setpriority(2)) and change the nice value for arbi‐\n                trary processes;\n              * set real-time scheduling policies for calling process, and set scheduling policies and\n                priorities   for   arbitrary   processes   (sched_setscheduler(2),  sched_setparam(2),\n                shed_setattr(2));\n              * set CPU affinity for arbitrary processes (sched_setaffinity(2));\n              * set I/O scheduling class and priority for arbitrary processes (ioprio_set(2));\n              * apply migrate_pages(2) to arbitrary processes and allow processes to  be  migrated  to\n                arbitrary nodes;\n              * apply move_pages(2) to arbitrary processes;\n              * use the MPOL_MF_MOVE_ALL flag with mbind(2) and move_pages(2).\n\nThanks,\nLukas."
					},
					{
						"isprivate": "0",
						"commentid": "11539495",
						"comment_count": "2",
						"who": {
							"text": "lsalgados",
							"name": "Luis Salgado Sandoval"
						},
						"bug_when": "2018-04-29 22:52:57 +0000",
						"thetext": "*** Bug 1573032 has been marked as a duplicate of this bug. ***"
					},
					{
						"isprivate": "0",
						"commentid": "11541246",
						"comment_count": "3",
						"who": {
							"text": "louzaoh",
							"name": "Héctor Louzao"
						},
						"bug_when": "2018-04-30 14:18:17 +0000",
						"thetext": "mysql-workbench-community-6.3.10-1.fc27.x86_64\nmysql-connector-python3-8.0.11-1.fc28.x86_64\nmysql-community-server-8.0.11-1.fc28.x86_64\nmysql-shell-8.0.11-1.fc28.x86_64\nmysql-community-client-8.0.11-1.fc28.x86_64\nmysql80-community-release-fc28-1.noarch\nmysql-community-libs-8.0.11-1.fc28.x86_64\nmysql-connector-java-8.0.11-1.fc28.noarch\nmysql-community-common-8.0.11-1.fc28.x86_64\n\nselinux-policy-3.14.1-21.fc28.noarch\nselinux-policy-targeted-3.14.1-21.fc28.noarch\n\nsetroubleshoot-3.3.17-1.fc28.x86_64\nsetroubleshoot-plugins-3.3.9-3.fc28.noarch\nsetroubleshoot-server-3.3.17-1.fc28.x86_64\n\nI'm in F28 beta now and with the inminent release of MYSQL 8 (GA) some days ago this still happen ....\n\nRegards.,"
					},
					{
						"isprivate": "0",
						"commentid": "11618581",
						"comment_count": "4",
						"who": {
							"text": "lvrabec",
							"name": "Lukas Vrabec"
						},
						"bug_when": "2018-05-21 13:08:47 +0000",
						"thetext": "*** Bug 1579160 has been marked as a duplicate of this bug. ***"
					},
					{
						"isprivate": "0",
						"commentid": "11671650",
						"comment_count": "5",
						"who": {
							"text": "hhorak",
							"name": "Honza Horak"
						},
						"bug_when": "2018-06-05 10:05:02 +0000",
						"thetext": "Looking at the packages carefully, those are coming from upstream, in Fedora we have community-mysql, so re-assigning. I think we should check whether the same issue is also seen in case of community-mysql packages."
					},
					{
						"isprivate": "0",
						"commentid": "11673418",
						"comment_count": "6",
						"who": {
							"text": "terje.rosten",
							"name": "Terje Røsten"
						},
						"bug_when": "2018-06-05 17:23:02 +0000",
						"thetext": "I think this is due to MySQL WL#9467 - Resource Groups\n\ncommit:\n https://github.com/mysql/mysql-server/commit/c47051b4be2110ed6225860448fe8657cf500a4a\n\nWorkLog:\n https://dev.mysql.com/worklog/task/?id=9467\n\nwhich is new in MySQL 8.0. \n\nThe move to /usr/bin and this sys_nice issue should be fixed in  selinux-policy\nI think?"
					},
					{
						"isprivate": "0",
						"commentid": "11673433",
						"comment_count": "7",
						"who": {
							"text": "terje.rosten",
							"name": "Terje Røsten"
						},
						"bug_when": "2018-06-05 17:25:31 +0000",
						"thetext": "FYI: move to /usr/bin/ is\n\n https://bugzilla.redhat.com/show_bug.cgi?id=1584643"
					},
					{
						"isprivate": "0",
						"commentid": "11965125",
						"comment_count": "8",
						"who": {
							"text": "jjanco",
							"name": "Jakub Jančo"
						},
						"bug_when": "2018-08-29 14:10:13 +0000",
						"thetext": "In downstream community-mysql we are missing cap_sys_nice on mysqld binary. Therefore we dont see selinux AVCs. But resource groups dont work.\n\nReproduce:\n# dnf install community-mysql-server\n# systemctl start mysql\nConnect to mysql server and create resource group and apply:\nmysql> CREATE RESOURCE GROUP rg1\n  TYPE = USER\n  VCPU = 0\n  THREAD_PRIORITY = 19;\nmysql> SET RESOURCE GROUP rg1;\n\nMaybe here should be some error message about \"there is not capability set on binary\". But currently I cant see any visible clue that groups dont work. Only thread nice doesnt change, while you are connected to mysql server, on mysql server run:\n# ps -T ax -o ni,cmd | grep mysqld | grep '^ 19 '\n<empty>\n\n# systemctl stop mysqld\nAfter you set capability on binary:\n# setcap cap_sys_nice+ep /usr/libexec/mysqld\n# systemctl start mysqld\nAVCs appear.\n\nNow if you try to set resource group in current login session:\nmysql> SET RESOURCE GROUP rg1;\n# ps -T ax -o ni,cmd | grep mysqld | grep '^ 19 '\n 19 /usr/libexec/mysqld --basedir=/usr\n\nSeems it works, but these log entries appears in mysql log:\n\n2018-08-29T13:30:08.980728Z 8 [ERROR] [MY-010985] [Server] Setting thread priority 0 to thread id 1099 failed. (error code 0 - Success).\n2018-08-29T13:30:08.980765Z 8 [Warning] [MY-010979] [Server] Unable to apply resource group controller USR_default.\n\nCreating and installing selinux module from audited errors allows setting resource groups without errors:\naudit2allow -a -M test\nsemodule -i test.pp"
					},
					{
						"isprivate": "0",
						"commentid": "11990647",
						"comment_count": "9",
						"who": {
							"text": "mschorm",
							"name": "Michal Schorm"
						},
						"bug_when": "2018-09-06 05:20:17 +0000",
						"thetext": "Fix pushed to the repo; rebuilds are on the way."
					},
					{
						"isprivate": "0",
						"commentid": "12067091",
						"comment_count": "10",
						"who": {
							"text": "louzaoh",
							"name": "Héctor Louzao"
						},
						"bug_when": "2018-10-01 14:47:22 +0000",
						"thetext": "finally works in F28 follow process :\n\nhttps://bugzilla.redhat.com/show_bug.cgi?id=1629380\n\nhttps://bugs.mysql.com/bug.php?id=92476\n\nThx fedora folks...\n\nregards\n\nmysql-connector-python3-8.0.12-1.fc28.x86_64\nmysql-connector-java-8.0.12-1.fc28.noarch\nmysql-community-client-8.0.12-1.fc28.x86_64\nmysql-community-common-8.0.12-1.fc28.x86_64\nmysql-workbench-community-8.0.12-1.fc28.x86_64\nmysql-connector-python-8.0.12-1.fc28.x86_64\nmysql-community-server-8.0.12-1.fc28.x86_64\nmysql-community-libs-8.0.12-1.fc28.x86_64\nmysql80-community-release-fc28-1.noarch\nmysql-shell-8.0.12-1.fc28.x86_64\n\nselinux-policy-targeted-3.14.1-42.fc28.noarch\nselinux-policy-3.14.1-42.fc28.noarch\nselinux-policy-devel-3.14.1-42.fc28.noarch\n\nsetroubleshoot-server-3.3.17-1.fc28.x86_64\nsetroubleshoot-plugins-3.3.9-3.fc28.noarch\nsetroubleshoot-3.3.17-1.fc28.x86_64"
					},
					{
						"isprivate": "0",
						"commentid": "12162719",
						"comment_count": "11",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2018-10-31 13:02:08 +0000",
						"thetext": "community-mysql-8.0.13-1.fc29 has been submitted as an update to Fedora 29. https://bodhi.fedoraproject.org/updates/FEDORA-2018-c82fc3e109"
					},
					{
						"isprivate": "0",
						"commentid": "12164244",
						"comment_count": "12",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2018-10-31 18:52:36 +0000",
						"thetext": "community-mysql-8.0.13-1.fc29 has been pushed to the Fedora 29 testing repository. If problems still persist, please make note of it in this bug report.\nSee https://fedoraproject.org/wiki/QA:Updates_Testing for\ninstructions on how to install test updates.\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2018-c82fc3e109"
					},
					{
						"isprivate": "0",
						"commentid": "12194565",
						"comment_count": "13",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2018-11-10 03:18:14 +0000",
						"thetext": "community-mysql-8.0.13-1.fc29 has been pushed to the Fedora 29 stable repository. If problems still persist, please make note of it in this bug report."
					}
				]
			}
		},
		{
			"bug_id": 1590369,
			"title": "Move log to /var/log/mysql/mysqld.log",
			"bugzilla": {
				"bug_id": "1590369",
				"creation_ts": "2018-06-12 13:57:30 +0000",
				"short_desc": "Move log to /var/log/mysql/mysqld.log",
				"delta_ts": "2018-11-10 03:18:27 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"priority": "unspecified",
				"bug_severity": "unspecified",
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "11696693",
						"comment_count": "0",
						"who": {
							"text": "hhorak",
							"name": "Honza Horak"
						},
						"bug_when": "2018-06-12 13:57:30 +0000",
						"thetext": "Description of problem:\nHaving the error log file in /var/log means that when missing, daemon (running as mysql:mysql) cannot re-create it. It makes troubles when rotating logs, or when an admin removes it manually. In that case, admin is responsible for re-creating the empty log file now, with proper owner and SELinux context.\n\nI'd like to propose moving the error log to a separate directory properly owned by mysql:mysql user, so daemon can re-created when missing. It's done similarly in mariadb already.\n\nThat obviously changes default behaviour, so we should only do it in F29 and further, and document it in Release Notes.. The expected changes are the following:\n\n* For users who have done some changes in /etc/my.cnf.d/community-mysql-server.cnf, the config will stay to use /var/log/mysqld.log\n* For users who have not done any changes in /etc/my.cnf.d/community-mysql-server.cnf, or install MySQL on a fresh machine, they will use /var/log/mysql/mysqld.log\n\nSo, the impact is not that the service would not start, it's just that some monitoring tools would need to re-configure."
					},
					{
						"isprivate": "0",
						"commentid": "11696697",
						"comment_count": "1",
						"who": {
							"text": "hhorak",
							"name": "Honza Horak"
						},
						"bug_when": "2018-06-12 13:58:36 +0000",
						"thetext": "Michal, Norvald, what do you think?"
					},
					{
						"isprivate": "0",
						"commentid": "11699380",
						"comment_count": "2",
						"who": {
							"text": "norvald.ryeng",
							"name": "Norvald H. Ryeng"
						},
						"bug_when": "2018-06-13 08:33:49 +0000",
						"thetext": "I agree. Moving the log file to /var/log/mysql/ makes sense."
					},
					{
						"isprivate": "0",
						"commentid": "11910157",
						"comment_count": "3",
						"who": {
							"text": "jkurik",
							"name": "Jan Kurik"
						},
						"bug_when": "2018-08-14 11:22:32 +0000",
						"thetext": "This bug appears to have been reported against 'rawhide' during the Fedora 29 development cycle.\nChanging version to '29'."
					},
					{
						"isprivate": "0",
						"commentid": "12162720",
						"comment_count": "4",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2018-10-31 13:02:16 +0000",
						"thetext": "community-mysql-8.0.13-1.fc29 has been submitted as an update to Fedora 29. https://bodhi.fedoraproject.org/updates/FEDORA-2018-c82fc3e109"
					},
					{
						"isprivate": "0",
						"commentid": "12164245",
						"comment_count": "5",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2018-10-31 18:52:44 +0000",
						"thetext": "community-mysql-8.0.13-1.fc29 has been pushed to the Fedora 29 testing repository. If problems still persist, please make note of it in this bug report.\nSee https://fedoraproject.org/wiki/QA:Updates_Testing for\ninstructions on how to install test updates.\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2018-c82fc3e109"
					},
					{
						"isprivate": "0",
						"commentid": "12194566",
						"comment_count": "6",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2018-11-10 03:18:27 +0000",
						"thetext": "community-mysql-8.0.13-1.fc29 has been pushed to the Fedora 29 stable repository. If problems still persist, please make note of it in this bug report."
					}
				]
			}
		}
	],
	"builds": [
		{
			"epoch": 0,
			"nvr": "community-mysql-8.0.13-1.fc29",
			"release_id": 23,
			"signed": true,
			"type": "rpm",
			"package": {
				"aarch64": [
					{
						"name": "community-mysql-common",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-server",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-libs",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-libs-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-devel",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-errmsg",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-test",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-devel-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-server-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-test-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-debugsource",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					},
					{
						"name": "community-mysql-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "aarch64"
					}
				],
				"armv7hl": [
					{
						"name": "community-mysql-devel-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-devel",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-errmsg",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-server",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-test",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-libs",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-server-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-libs-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-common",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-test-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					},
					{
						"name": "community-mysql-debugsource",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "armv7hl"
					}
				],
				"i686": [
					{
						"name": "community-mysql-devel",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-common",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-test-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-errmsg",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-debugsource",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-libs",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-devel-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-server",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-server-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-libs-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					},
					{
						"name": "community-mysql-test",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "i686"
					}
				],
				"ppc64le": [
					{
						"name": "community-mysql-devel-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-server-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-devel",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-test-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-errmsg",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-debugsource",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-common",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-test",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-libs-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-libs",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					},
					{
						"name": "community-mysql-server",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "ppc64le"
					}
				],
				"s390x": [
					{
						"name": "community-mysql",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-libs",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-common",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-errmsg",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-server",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-devel",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-test",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-debugsource",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-libs-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-server-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-devel-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					},
					{
						"name": "community-mysql-test-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "s390x"
					}
				],
				"src": [
					{
						"name": "community-mysql",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "src"
					}
				],
				"x86_64": [
					{
						"name": "community-mysql-common",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-devel-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-server-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-test-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-libs-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-devel",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-server",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-debugsource",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-errmsg",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-test",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-debuginfo",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					},
					{
						"name": "community-mysql-libs",
						"version": "8.0.13",
						"release": "1.fc29",
						"arch": "x86_64"
					}
				]
			}
		}
	],
	"close_bugs": true,
	"comments": [
		{
			"id": 855303,
			"text": "This update has been submitted for testing by mschorm. ",
			"timestamp": "2018-10-31 13:02:04",
			"update_id": 125710,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 855742,
			"text": "This update has been pushed to testing.",
			"timestamp": "2018-10-31 18:54:12",
			"update_id": 125710,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 858812,
			"text": "This update has reached 7 days in testing and can be pushed to stable now if the maintainer wishes",
			"timestamp": "2018-11-08 00:01:19",
			"update_id": 125710,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 859100,
			"text": "This update has been submitted for batched by mschorm. ",
			"timestamp": "2018-11-08 12:55:35",
			"update_id": 125710,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 859362,
			"text": "This update has been submitted for stable by bodhi. ",
			"timestamp": "2018-11-09 03:01:38",
			"update_id": 125710,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 859876,
			"text": "This update has been pushed to stable.",
			"timestamp": "2018-11-10 03:19:44",
			"update_id": 125710,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		}
	],
	"content_type": "rpm",
	"date_pushed": "2018-11-10 03:16:10",
	"date_stable": "2018-11-10 03:16:10",
	"date_submitted": "2018-10-31 13:02:04",
	"date_testing": "2018-10-31 18:48:05",
	"meets_testing_requirements": true,
	"notes": "**MySQL 8.0.13**\n\nRelease notes:\n\n    https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-13.html\n\nCVEs fixed:\n\n    CVE-2018-3276 CVE-2018-3200 CVE-2018-3137 CVE-2018-3284 CVE-2018-3195\n    CVE-2018-3173 CVE-2018-3212 CVE-2018-3279 CVE-2018-3162 CVE-2018-3247\n    CVE-2018-3156 CVE-2018-3161 CVE-2018-3278 CVE-2018-3174 CVE-2018-3282\n    CVE-2018-3285 CVE-2018-3187 CVE-2018-3277 CVE-2018-3144 CVE-2018-3145\n    CVE-2018-3170 CVE-2018-3186 CVE-2018-3182 CVE-2018-3133 CVE-2018-3143\n    CVE-2018-3283 CVE-2018-3171 CVE-2018-3251 CVE-2018-3286 CVE-2018-3185\n    CVE-2018-3280 CVE-2018-3203 CVE-2018-3155\n\nMaintainer's notes:\n\n    MySQL now builds MySQL Router (for innoDB cluster) as a part of the server. However since it can't be used without MySQL Shell, whuch is not packed to Fedora, nor the Router is allowed to be built & packed.",
	"pushed": true,
	"release": {
		"branch": "f29",
		"candidate_tag": "f29-updates-candidate",
		"composed_by_bodhi": true,
		"create_automatic_updates": false,
		"dist_tag": "f29",
		"id_prefix": "FEDORA",
		"long_name": "Fedora 29",
		"mail_template": "fedora_errata_template",
		"name": "F29",
		"override_tag": "f29-override",
		"package_manager": "dnf",
		"pending_signing_tag": "f29-signing-pending",
		"pending_stable_tag": "f29-updates-pending",
		"pending_testing_tag": "f29-updates-testing-pending",
		"stable_tag": "f29-updates",
		"state": "archived",
		"testing_repository": "updates-testing",
		"testing_tag": "f29-updates-testing",
		"version": "29"
	},
	"severity": "medium",
	"stable_karma": 3,
	"status": "stable",
	"suggest": "unspecified",
	"test_gating_status": "ignored",
	"title": "community-mysql-8.0.13-1.fc29",
	"type": "security",
	"url": "https://bodhi.fedoraproject.org/updates/FEDORA-2018-c82fc3e109",
	"unstable_karma": -1,
	"updateid": "FEDORA-2018-c82fc3e109",
	"user": {
		"avatar": "https://seccdn.libravatar.org/avatar/4cb3ac6974b562445771614a80b71a2ee8699042757acf98e9c094ea88b1821e?s=24&d=retro",
		"email": "mschorm@redhat.com",
		"groups": [
			{
				"name": "packager"
			},
			{
				"name": "ipausers"
			},
			{
				"name": "fedora-contributor"
			},
			{
				"name": "signed_fpca"
			},
			{
				"name": "fedorabugs"
			},
			{
				"name": "db-sig"
			}
		],
		"id": 3283,
		"name": "mschorm",
		"openid": "mschorm.id.fedoraproject.org"
	},
	"version_hash": "42a256de45f816812b5680aa7ca2d91fa3941d90"
}
